manifests/config.pp

```diff
@@ -3,19 +3,22 @@
 # This class is called from autosign for service config.
 #
 class autosign::config {
-  $config_ensure = $::autosign::ensure ? {
+  $_config_ensure = $autosign::ensure ? {
```

Comment:
why omit the explicit top-scope namespacing here, going from `$::autosign::ensure` to plain `$autosign::ensure`? I'm not opposed, I'd just like to know if the convention here has changed.

Reply:
The docs no longer use top scope for anything other than classes inclusion / declarations.
https://puppet.com/docs/puppet/5.3/hiera_migrate_modules.html

The addition of the `_` prefix was a result of https://puppet.com/docs/puppet/5.0/lang_variables.html#naming